### PR TITLE
upgrade all projects to .NET 4.6

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

File: src/Dashboard/Web.config [hunk 1]
```diff
@@ -1,100 +1,107 @@
...
+      </system.Web>
+  -->
   <system.web>
-    <customErrors mode="RemoteOnly" />
-    <compilation debug="true" targetFramework="4.5" />
-    <httpRuntime targetFramework="4.5" />
-    <authentication mode="None" />
+    <customErrors mode="RemoteOnly"/>
+    <compilation debug="true" targetFramework="4.6"/>
+    <httpRuntime targetFramework="4.5"/>
```
Comment (mathewc) [inline src/Dashboard/Web.config]: This stays 4.5?

**Comment by mathewc** [comment]
Are you going to do the same for the Extensions repo?

File: src/Dashboard/Web.config (hunk 1 above)
Comment (brettsam) [inline src/Dashboard/Web.config]: It auto-added the runtime change above in a comment. I didn't want to do any manual adjusting here without a great way to test it out. My plan was just to leave it this way, figuring we could update it later.

**Comment by brettsam** [comment]
@mathewc -- yeah, i'm doing that now. Pulling this in and moving everything to 4.6

**Pull request merged**